.vuepress/_redirects

```diff
@@ -155,3 +155,6 @@
 /addons/persistence/mysql/ https://v2.openhab.org/addons/persistence/mysql/ 301
 /addons/persistence/sitewhere/ https://v2.openhab.org/addons/persistence/sitewhere/ 301
 /addons/persistence/cosm/ https://v2.openhab.org/addons/persistence/cosm/ 301
+
+## Redirect all jdbc persistence documentation to main jdbc page with wildcard
+/addons/persistence/jdbc-*/ /addons/persistence/jdbc/
```

Comment:
According to the netlify docs (https://docs.netlify.com/routing/redirects/redirect-options/#splats), the asterisk must be at the end of the path:

```suggestion
/addons/persistence/jdbc-* /addons/persistence/jdbc/
```

I would also personally prefer to put that into line 45 and not at the end.